app/src/main/res/xml/preferences.xml

```diff
@@ -44,6 +44,10 @@
             android:title="@string/statistics_label"
             android:icon="?attr/statistics" />
 
+    <Preference
+            android:key="notification"
```

Comment:
Please use plural here and move the setting above the statistics screen. Also, having an icon would be great :) I think the bell icon that is already shipped with Android Studio would work (new->Vector asset).